packages/web/app/src/components/target/history/labelize.tsx

```diff
@@ -0,0 +1,10 @@
+import { ReactNode } from 'react';
+import reactStringReplace from 'react-string-replace';
+import { Label } from '@/components/common';
+
+export function labelize(message: string): ReactNode[] {
+  // Replace '...' with <Label>...</Label>
+  return reactStringReplace(message, /'([^']+)'/gim, (match, i) => (
+    <Label key={i} dangerouslySetInnerHTML={{ __html: match }} />
```

Comment:
I think better to use `dangerouslySetInnerHTML` instead of `.replaceAll('&apos;', "'").replaceAll('&quot;', '"')`

Reply:
My concern is that using `dangerouslySetInnerHTML` will inject things we do not expect. 
Let's say someone is using some HTML or malicious syntax in the schema description, it might be injected all the way to the frontend?

Reply:
you are right, I totally forgot about this case, I will push a fix